src/ensembl/src/content/app/custom-download/containers/content/filter-accordion/FiltersAccordion.scss

```diff
@@ -34,3 +68,7 @@
   top: 10px;
```

Comment:
What does the `tempPadding` rule above mean?  Why is it `temp`?

Reply:
It's to add padding to the temporary text that is displayed. Renamed it to `temporaryContentPadding`.